translator/tocwconfig/sampleConfig/complete_linux_config.conf

```diff
@@ -9,7 +9,7 @@
   logtarget = "lumberjack"
   metric_batch_size = 1000
   metric_buffer_limit = 10000
-  omit_hostname = false
+  omit_hostname = true
```

Comment:
what is omit hostname for? is there a reason we need it in the toml?

Reply:
This is because there is hostname being queried for the jmx metrics to put in the yaml. This makes sure we do not do that. It is non deterministic for unit test code. For the telegraf metrics it does the hostname look up only at runtime and not translator time. That is why I added this here.